src/AutoUI/schemaOps.ts

```diff
@@ -142,3 +143,30 @@ export const getFieldForFormat = (schema: JSONSchema, format: string) => {
 
 	return propertyKeyWithFormat;
 };
+
+export const convertRefSchemeToSchemaPath = (refScheme: string | undefined) => {
+	return !!refScheme
+		? refScheme
+				.split('.')
+				.join('.properties.')
+				.replace(/\[\d+\]/g, '.items')
+		: // TODO: This atm doesn't support ['my property']
+		  refScheme;
```

Comment:
```suggestion
        // TODO: This atm doesn't support ['my property']
	return refScheme?
				.split('.')
				.join('.properties.')
				.replace(/\[\d+\]/g, '.items');
```

Reply:
Looks like you fixed this in one place but not this place. Which raises the question of why we have 2 instances of this function?